src/main/java/com/amazon/opendistro/elasticsearch/performanceanalyzer/rca/store/rca/HighHeapUsageClusterRca.java

```diff
@@ -49,7 +49,7 @@
  */
 public class HighHeapUsageClusterRca extends Rca<HighHeapUsageClusterFlowUnit> {
 
-  private static final Logger LOG = LogManager.getLogger(HighHeapUsageRca.class);
+  private static final Logger LOG = LogManager.getLogger(HighHeapUsageOldGenRca.class);
```

Comment:
Did you mean `HighHeapUsageClusterRca.class` instead of `HighHeapUsageOldGenRca.class`?

Reply:
No, I am trying to replace HighHeapUsageRca with HighHeapUsageOldGenRca because we have both young gen and old gen RCAs.Since we are adding HighHeapUsageYoungGenRca in this PR so it would be better to change the HighHeapUsageRca as well.

Reply:
Discussed offline.